addons/repair/tests/test_repair.py

```diff
@@ -51,17 +51,17 @@ def setUpClass(cls):
         cls.product_consu_order_repair = cls.env['product.product'].create({
             'name': 'Repair Consumable',
             'type': 'consu',
-            'create_repair': True,
```

Comment:
Since we remove the possibility for non-`service` products to generate repair orders when sold, this had no meaning anymore. Same thing for `product_storable_order_repair`.

The tests where these were used *may* not be relevant anymore, as it's no longer a supported case. You'll need to check if they would still make sense with services generating the Repair Orders or not.